added few variables for criminality

RemiCSK opened the pull request:
added bruglary, assault,otheraussault and destruction for the year 2019. There are from 45% to 70% percent of NaN so we have to be carefull.

Don't forget to create a local folder and put the file in it.

RemiCSK committed:
added few variables for criminality

added bruglary, assault,otheraussault and destruction for the year 2019.
There are from 45% to 70% percent of NaN so we have to be carefull.

Don't forget to create a local folder and put the file in it.

RemiCSK committed:
Update controlvariables.py

bug correction, now using excel file instead of csv because csv was missing values for Lyon. The excel file is glitchy, you may need to register it to run the code.

pwisniewski-econ committed:
Merge branch 'main' into criminality-control-variables

pwisEcon commented:
Good job, but be careful about changes in main with regards to your branches as it can create conflicts. Always try to be up-to-date with main.

RemiCSK committed:
Update controlvariables.py

RemiCSK committed:
optimized file for criminality

pwisniewski-econ committed:
Fill NaN with complementinfotaux

Fill NaN with complementinfotaux and change from absolute value to incidence of crimes

pwisniewski-econ commented on src/controlvariables.py:
I added what we discussed previously (ie. transforming NaN crime rates to the departmental average, not perfect but better than setting NaNs to 0 or creating categorical dummies).

pwisniewski-econ reviewed (approved):
Approved! 👍
